client/src/Components/Feature/Feature.tsx

```diff
@@ -3,13 +3,23 @@ import { useFeatureFlags } from '@/Components/FeatureFlagsContext/FeatureFlagsCo
 
 interface FeatureProps {
   flag: string;
+  invert: boolean | string;
   children: ReactNode;
 }
 
-const Feature = ({ flag, children }: FeatureProps) => {
+const Feature = ({ flag, invert, children }: FeatureProps) => {
```

Comment:
You should be able to pass in invert = false as a default here

Reply:
Awesome, thanks for pointing that out!